### PR TITLE
PUD-1398 warrantReferenceNumber String property

### PR DESCRIPTION
**Pull request by AndyNewton**
PUD-1398  BE: Store and update new warrant reference number property on Recall

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallController.kt [hunk 1]
```diff
@@ -330,7 +331,8 @@ data class RecallResponse(
   val dossierCreatedByUserName: FullName? = null,
   val lastKnownAddressOption: LastKnownAddressOption? = null,
   val arrestIssues: Boolean? = null,
-  val arrestIssuesDetail: String? = null
+  val arrestIssuesDetail: String? = null,
+  val warrantReferenceNumber: String? = null,
```
Comment (philsegal) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallController.kt:335]: is it worth making it a tiny type? :)

**Review by philsegal — APPROVED** [review]
personally, I'd make it a tiny type...

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/SpringFoxConfig.kt [hunk 2]
```diff
@@ -82,6 +85,7 @@ class SpringFoxConfig {
       .directModelSubstitute(PoliceForceName::class.java, String::class.java)
       .directModelSubstitute(FieldName::class.java, String::class.java)
       .directModelSubstitute(FieldPath::class.java, String::class.java)
+      .directModelSubstitute(ColumnName::class.java, String::class.java)
```
Comment (AndyNewton) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/SpringFoxConfig.kt:88]: @philsegal :  I noticed this was not present so added.

File: scripts/getClientToken.sh [hunk 3]
```diff
@@ -1,18 +1,22 @@
 #!/bin/bash
 
 # With a valid hmpps-auth client_id and client_secret this script will create a temporary token for use
-# as a `Bearer` token via e.g. Postman or Swagger.
+# as a `Bearer` token via e.g. Postman or Swagger (use "Bearer <token>").
```
Comment (AndyNewton) [inline scripts/getClientToken.sh:4]: Not related - but updates to comments in this script only.

**Review by philsegal — APPROVED** [review]

**Pull request merged**